### DISCUSSION TITLE
Error 500

### DISCUSSION THREAD

**Discussion by HadhoumH** [community/community#51114] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I commit change, I got this error: 
Looks like something went wrong! We track these errors automatically, but if the problem persists feel free to contact us. In the meantime, try refreshing.

**Comment by GyulyVGC** [comment]
It's a problem on GitHub side: see [here](https://www.githubstatus.com) @HadhoumH 
We just have to wait and hopefully it'll be solved soon.
Don't forget to mark this comment as answer so that other people can view this discussion as solved! 

![Screenshot 2023-03-27 at 15 05 15](https://user-images.githubusercontent.com/100347457/227947004-184645b4-1590-40d8-afe0-825a878b6934.png)

**Comment by PajaritoMoyqi** [comment]
Things are better now than before.

**Comment by HadhoumH** [comment]
Thanks!
The issue is solved
